src/Header/Logger.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_SRC_LOGGER_H_
+#define GROUP4_SRC_LOGGER_H_
+
+#include <string>
+#include <memory>
+#include "OperatorData.h"
+
+// Write operation and result log
+class Logger
+{
+public:
+    // Singleton code convention method
+    static Logger* GetInstance();
+    
+    // Write data and warning message
+    void WriteLog(const OperatorData &data, const Level &level) const;
```

Comment:
Where is enum Level defined? Usually we define it in the same file or in the same class.